views/Graph.react.js

```diff
@@ -214,48 +240,71 @@ class Graph extends Component {
   handleResize = () => {
     this.setState({
       graphWidth: window.innerWidth,
-      graphHeight: window.innerHeight - 30
+      graphHeight: window.innerHeight - 30,
     })
   }
   resync = () => {
-    window.fetch(this.state.socketIOSubjectRoom, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      credentials: 'same-origin'
-    }).then((response) => {
-      if (response.status !== 201) {
-        return
-      }
-      return response.json()
-    }).then((response) => {
-      this.setState({ taskId: response.correlationId, loading: true, success: false })
-    })
+    window
+      .fetch(this.state.socketIOSubjectRoom, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'same-origin',
+      })
+      .then((response) => {
+        if (response.status !== 201) {
+          return
+        }
+        return response.json()
+      })
+      .then((response) => {
+        this.setState({
+          taskId: response.correlationId,
+          loading: true,
+          success: false,
+        })
+      })
   }
   closeStat = () => {
     this.setState({
       openStat: false,
-    });
+    })
   }
   openStat = () => {
     this.setState({
       openStat: true,
-    });
-  }
-  componentDidUpdate() {
+    })
   }
+  componentDidUpdate() {}
   // eslint-disable-next-line react/no-deprecated
   async componentWillMount() {
```

Comment:
might be out of scope of this PR but I believe we'll want to load this data on `componentDidMount`. I think React suggests not doing this in `componentWillMount`.

Reply:
What do you think about transposing this to a functional component, a more modern approach to the component. Reuse some of the components that we've added like the AppLayout component, Config.react.js, Main.react.js, Graph.react.js etc... they all call the same api's like fetch configurations, but each of those functions are repeated in all of those pages. So there's lot's of opportunities.

On the backend I can split up the index.js file and separate the routes to be individual routers, like I did with the charts.js file.

Reply:
I think those are all great ideas, but out of scope of this PR. Worth making a ticket for the refactor, though 😄